client/router_create.go

```diff
@@ -1189,7 +1195,11 @@ sasldb_path: /tmp/skrouterd.sasldb
 					log.Printf("Failed to retrieve route %q: %s", types.ClaimRedemptionRouteName, err.Error())
 				}
 			} else if options.Spec.IsIngressLoadBalancer() {
-				err = cli.appendLoadBalancerHostOrIp(types.ControllerServiceName, van.Namespace, &cred)
+				var timeout = loadbalancertimeout
+				if timeout <= 0 {
+					timeout = types.DefaultTimeout
+				}
```

Comment:
Same here, if you set the timeout into the `options SiteConfig` variable, you won't need this.